themes/ropensci/layouts/partials/auto-syntax.html

```diff
@@ -1,7 +1,8 @@
  <!-- loads scripts for highlight JS -->
     <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
-
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
```

Comment:
this is due to #622 -- maybe I should add this on master and rebase but for now it's in this PR.